packages/jest-runtime/src/index.ts

```diff
@@ -211,10 +219,10 @@ class Runtime {
       config.setupFiles.forEach(filePath => {
         if (filePath && filePath.includes(NODE_MODULES)) {
           const moduleID = this._resolver.getModuleID(
-            this._virtualMocks,
+            fromEntries(this._virtualMocks),
```

Comment:
let's call `fromEntries` once before the loop, no need to recalculate it

Reply:
oh, didn't notice this was in a loop. Good call, will do